dozer-orchestrator/src/simple/orchestrator.rs

```diff
@@ -45,17 +47,27 @@ use tokio::sync::{broadcast, oneshot};
 pub struct SimpleOrchestrator {
     pub config: Config,
     pub cache_manager_options: CacheManagerOptions,
+    pub executor_options: ExecutorOptions,
```

Comment:
Looks like calculated data. Maybe calculate it when needed instead of adding a member?

Reply:
Can you give me more details on this comment?
These parameters are coming from the configuration file

Reply:
Exactly. These parameters are coming from the configuration file so I prefer not storing it as a field of the struct, and creating a method `fn get_executor_options(config: &Config) -> ExecutorOptions` instead. But that's just me and I'm not sure if this approach makes sense to you.